[0;31mInput directory[0m
/AB_20T_input/dataset/subset_23G_CliveOME_5mc_dataset_POD5/
[0;31mSave path[0m
/AB_20T_output/nanopore_output/benchmark_run_output/
[0;31mName of this run[0m
dorado_hac_5mCG_5hmCG_13-12_benchmark_gpu004
[0;31mlog path[0m
/AB_20T_output/nanopore_output/benchmark_run_logs/dorado_hac_5mCG_5hmCG_13-12_benchmark_gpu004
Start run
Select interface
ibp216s0
Select host (nfs01.ib)
nfs01.ib
Output file: /AB_20T_output/nanopore_output/benchmark_run_logs/dorado_hac_5mCG_5hmCG_13-12_benchmark_gpu004/connection_log_dorado_hac_5mCG_5hmCG_13-12_benchmark_gpu004.csv
13:12:39:  [0;31mNo communication with nfs01.ib found.[0m
terminate called after throwing an instance of 'std::runtime_error'
  what():  unknown simplex model /u/dssc/tolloi/dorado_0.4.0_pre_built/bin/models/dna_r10.4.1_e8.2_400bps_hac@v4.1.0
13:12:40:  [0;31mNo communication with nfs01.ib found.[0m
13:12:41:  [0;31mNo communication with nfs01.ib found.[0m
13:12:42:  [0;31mNo communication with nfs01.ib found.[0m
./dorado_5mCG_5hmCG_benchmark_GPU.sh: line 51: 227179 Aborted                 (core dumped) dorado basecaller /u/dssc/tolloi/dorado_0.4.0_pre_built/bin/models/dna_r10.4.1_e8.2_400bps_hac@v4.1.0 $1/ -x cuda:0 --modified-bases 5mCG_5hmCG > $2/output_$run_name.bam
Traceback (most recent call last):
  File "/u/dssc/tolloi//Cluster_Basecalling_Manager/utility/gpu_average.py", line 2, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
Traceback (most recent call last):
  File "/u/dssc/tolloi//Cluster_Basecalling_Manager/utility/net_average.py", line 1, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[0;32m---------------------------------[0m
